src/Rector/Deprecation/FileBuildUrlRector.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace DrupalRector\Rector\Deprecation;
+
+use PhpParser\Node;
+use Rector\Core\Rector\AbstractRector;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+final class FileBuildUrlRector extends AbstractRector
```

Comment:
Shouldn't this be FileBuildUriRector not FileBuildUrlRector

Reply:
My bad, yes